model_card_toolkit/base_model_card_field.py

```diff
@@ -55,7 +59,7 @@ def to_proto(self) -> message.Message:
     for field_name, field_value in self.__dict__.items():
       if not hasattr(proto, field_name):
         raise ValueError(
-            "%s has no such field named '%s'." % (type(proto), field_name)
+            '%s has no such field named "%s".' % (type(proto), field_name)
```

Comment:
Is the swap from `"` to `'` due to an editor config from your side? Should we define one way in `yapf` to keep it consistent?

Reply:
No, it's not from an editor config: mct mostly uses single quotes, and I default to single quotes here like in other tf projects. I switched to single here when yapf complained about the mixing, but I didn't update all other files.

Great idea! Yes, let's define single quotes in yapf.

Reply:
Created https://github.com/tensorflow/model-card-toolkit/issues/279